yt_dlp/extractor/youtube/_base.py

```diff
@@ -472,8 +474,9 @@ def _initialize_pref(self):
         self._set_cookie('.youtube.com', name='PREF', value=urllib.parse.urlencode(pref))
 
     def _initialize_cookie_auth(self):
-        yt_sapisid, yt_1psapisid, yt_3psapisid = self._get_sid_cookies()
-        if yt_sapisid or yt_1psapisid or yt_3psapisid:
+        self._had_cookie_auth = False
+        if self._has_auth_cookies:
+            self._had_cookie_auth = True
```

Comment:
I'm fine keeping it as-is but what do you think about `_passed_auth_cookies` as a name instead of `_had_cookie_auth`?

Reply:
sure